src/memphis.ts

```diff
@@ -369,6 +372,56 @@ export class Memphis {
         }
     }
 
+    /**
+     * Attaches a schema to existing station.
+     * @param {String} name - schema name.
+     * @param {String} stationName - station name to attach schema to.
+     */
+    async attachSchema({ name, stationName }: { name: string; stationName: string }): Promise<void> {
+        try {
+            if (!this.isConnectionActive) throw new Error('Connection is dead');
+            if (name === '' || stationName === '') {
+                throw new Error('attachSchema error: Please provide all parameters');
```

Comment:
this is an error the user will catch so no need the function name in the error message - change to name and station name can not be empty